saq/worker.py

```diff
@@ -1,16 +1,28 @@
+from __future__ import annotations
+
 import asyncio
 import contextvars
 import logging
+import os
 import signal
 import traceback
-import os
+import typing as t
 
 from croniter import croniter
 
 from saq.job import Status
 from saq.queue import Queue
 from saq.utils import millis, now, seconds
 
+if t.TYPE_CHECKING:
+    from asyncio import Task
+    from collections.abc import Callable, Collection, Coroutine
```

Comment:
Are you happy for individual imports for `collections.abc`? Or prefer `from collections import abc` and `abc.Callable[...]` etc?

Reply:
it’s ok